lib/metrics/recorders/database.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2024 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const { DB, ALL } = require('../names')
+const { DESTINATIONS } = require('../../config/attribute-filter')
+
+/**
+ * @this ParsedStatement
+ * @param {TraceSegment}  segment - The segment being recorded.
+ * @param {string}        [scope] - The scope of the segment.
+ */
+
+function _recordMetrics(segment, scope) {
```

Comment:
nit but this doesn't have to be `_recordMetrics`. could just be `recordDbMetrics`